### PR TITLE
Feat: Added encryption on token

### PR DESCRIPTION
**Pull request by HaroldCuico**
* Added encryption to token before passing to URL
* Added redirection to login

### CODE REVIEW

File: WebApp/frontend/quiz-master/app/dashboard/game.tsx
Comment (jaymar921) [inline WebApp/frontend/quiz-master/app/dashboard/game.tsx:1]: Good implementation

File: WebApp/frontend/quiz_session/app/auth/page.js
Comment (jaymar921) [inline WebApp/frontend/quiz_session/app/auth/page.js:1]: I'll approve this but next time we need to make the  login url to dynamic, we don't know on what domain will be the production version will be

File: WebApp/frontend/quiz_session/app/components/welcome.js
Comment (jaymar921) [inline WebApp/frontend/quiz_session/app/components/welcome.js:1]: Nice 👍

**Review by jaymar921 — APPROVED** [review]
Good job 💯👍

**Review by kinloveko — APPROVED** [review]
Looks good!

**Pull request merged**